src/Home.js

```diff
@@ -0,0 +1,351 @@
+      case '04:30 PM':
+        return 'PM430start';
+      case '05:00 PM':
+        return 'PM5start';
+      case '05:30 PM':
+        return 'PM530start';
+      case '06:00 PM':
+        return 'PM6start';
+      case '06:30 PM':
+        return 'PM630start';
+      case '07:00 PM':
+        return 'PM7start';
+      case '07:30 PM':
+        return 'PM730start';
+      case '08:00 PM':
+        return 'PM8start';
+      case '08:30 PM':
+        return 'PM830start';
+      case '09:00 PM':
+        return 'PM9start';
+      case '09:30 PM':
+        return 'PM930start';
+      case '10:00 PM':
+        return 'PM10start';
+      default:
+        return;
+    }
+  }
+  
+  // contains the displayed list of events
+  const CalendarEvents = () => {
+    const gridLines = [];
+  
+    // Background Grid
+    for (let row = 1; row <= getCSSVar('--gridRowCount'); row++) {
+      for (let col = 1; col <= getCSSVar('--gridColCount'); col++) {
+        gridLines.push(
+          <div className='gridLine' key={`${row}-${col}`} style={{gridRow: row, gridColumn: col}} div/>
+        )
+      }
+    }
+  
+    return (
+      <section className='calendarGrid'>
+        {EVENTS.map((event) => {
+            return (
+              <Event name={event.name} 
+                    start={event.start} 
+                    end={event.end} 
+                    location={event.location} 
+                    sideColor={event.sideColor} 
+                    bgColor={event.bgColor}
+                    otherParty={event.otherParty}
+                    initials1={event.initials1}
+                    iconColor1={event.iconColor1}
+              />
+            )
+          })}
+        {gridLines}
+      </section>
+    )
+  }
+  // defines how an event is displayed
+  function Event(props) {
+    const startTime = props.start.toLocaleString('default', {hour: '2-digit', minute: '2-digit'});
+  
+    return (
+      <div className='eventContainer' style={{gridColumn: getGridCol(props.start), 
+                                                          gridRow: `${getGridRow(props.start)} / ${getGridRow(props.end)}`, 
+                                                          backgroundColor: props.bgColor}}>
+        {/* Side Bar */}
+        <div className='rect' style={{backgroundColor: props.sideColor}} />
+        {/* Event Info */}
+        <div className='eventInfo'>
+          {/* Event Top Bar */}
+          <div className='eventTopBar'>
+            {/* Time */}
+            <div className='eventTime'>{startTime}</div>
+            {/* Person Involved (if applicable) */}
+            {props.otherParty && 
+              <button className='userIcon calendarVariation' style={{backgroundColor: props.iconColor1}}>
+                {props.initials1}
+              </button>} 
+          </div>
+          {/* Event Name */}
+          <div className='eventName'>{props.name}</div>
+          {/* Event Location */}
+          {props.location}
```

Comment:
Notice how before every div here you have a comment describing what the div is for?
This is a great case for creating components. If you turned every one of these divs into it's own component, you code here could look like
```
<SideBar>
  <EventInfo>
    <EventTopBar>
      <EventTime />
      <PersonInvolved />
    </EventTopBar>
    <EventName />
    <EventLocation />
  </EventInfo>
</SideBar>
```
Look how clean!!!!
To do this, you'll need to use `props.children`. Here's a [guide](https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891), but in short you just need to define `<SideBar>` to return something like
```
// from within SideBar component
return (
  <div className='rect' .....>
    {props.children}
  </div>
)
```